ansible/roles/celery/tasks/main.yml

```diff
@@ -25,6 +44,13 @@
   sudo: yes
   tags: celery
 
+- name: Start docker daemon
```

Comment:
This should be a handler, which restarts the ``docker`` service, then add ``notify:`` to each of the other tasks that modify the Docker package.